### ISSUE TITLE
query type="search"

### ISSUE THREAD

**Issue by tetuaoro** [Baptistemontan/leptos_i18n#136]
# Description

Seems make a query when using `<I18nRoute />` breaks queries. The following code, if I hit _search_ button, the url become `/taoq=...` instead of `/tao?q=...`.

```rust
<form action="/tao">
    <input type="search" name="q" id="tao"/>
    <button type="submit">"Search"</button>
</form>
```

### Cargo.toml

```toml
leptos = { version = "0.6", features = ["experimental-islands"] }
leptos_i18n = { version = "0.4.0", default-features = false, features = ["cookie", "experimental-islands", "track_locale_files", "yaml_files"] }
```

### Fixed

It fixed if I remove `I18nRoute` or if I use the default lang. If I use another lang, it breaks.

**Comment by tetuaoro** [comment]
@Baptistemontan thank you, it fixed. Because v`0.5` is beta, I wrote the same in v`0.4`.

**Comment by Baptistemontan** [comment]
just published v0.4.1 with the fix
